.github/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+# Description
+
+Please include a summary of changes, motivation and context for this PR.
+
+Fixes # (issue)
+
+## Type of change
+
+Please tick all options which are relevant.
+
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Added/removed dependencies
+- [ ] Requires documentation updates
+
+# Testing
+
+Please describe the test(s) that you added and ran to verify your changes.
```

Comment:
The test should be self descriptive, or commented, I don't see the point in duplicating the effort in the PR description.